### ISSUE TITLE
Error when trying to deploy TCP Monitor without send or receive attributes

### ISSUE THREAD

**Issue by 0xHiteshPatel** [F5Networks/f5-appsvcs-extension#436]
### Environment
 * Application Services Version: 3.26.0 build 5
 * BIG-IP Version: 13.1.1.2.0.0.4

### Summary
According to the schema reference docs the `send` and `receive` attributes are not required when deploying a monitor with `monitorType` of `tcp`.  When trying to deploy without those attributes I receive an error as shown below.  I can workaround the issue by adding those attributes with empty strings, however the behavior is inconsistent not ideal and inconsistent with the docs.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "TCPTest": {
        "class": "Tenant",
        "TCP_Monitor_Test": {
            "class": "Application",
            "template": "generic",
            "TCP_Monitor": {
                "class": "Monitor",
                "monitorType": "tcp"
            },
            "TCP_Pool": {
                "class": "Pool",
                "monitors": [
                    {"use":"TCP_Monitor"}
                ],
                "members": [
                    {
                        "enable": true,
                        "adminState": "enable",
                        "shareNodes": true,
                        "serverAddresses": [
                            "1.1.1.1"
                        ],
                        "servicePort": 1234
                    }
                ],
                "loadBalancingMode": "least-connections-member"
            }
        }
    },
    "class": "ADC",
    "schemaVersion": "3.7.0",
    "id": "dda23da4-3f9b-49c3-91e8-aa6a3f88baa5",
    "updateMode": "selective",
    "controls": {
        "archiveTimestamp": "2021-03-23T17:14:46.030Z"
    }
}
```

2. Observe the following error response:
```json
{
    "code": 422,
    "errors": [
        "/TCPTest/TCP_Monitor_Test/TCP_Monitor: should have required property 'send'"
    ],
    "declarationFullId": "",
    "message": "declaration is invalid"
}
```

### Expected Behavior
This declaration should be valid

### Actual Behavior
This declaration is not valid

### Workaround
```json
{
    "TCPTest": {
        "class": "Tenant",
        "TCP_Monitor_Test": {
            "class": "Application",
            "template": "generic",
            "TCP_Monitor": {
                "class": "Monitor",
                "monitorType": "tcp",
                "send": "",
                "receive": ""
            },
            "TCP_Pool": {
                "class": "Pool",
                "monitors": [
                    {"use":"TCP_Monitor"}
                ],
                "members": [
                    {
                        "enable": true,
                        "adminState": "enable",
                        "shareNodes": true,
                        "serverAddresses": [
                            "1.1.1.1"
                        ],
                        "servicePort": 1234
                    }
                ],
                "loadBalancingMode": "least-connections-member"
            }
        }
    },
    "class": "ADC",
    "schemaVersion": "3.7.0",
    "id": "dda23da4-3f9b-49c3-91e8-aa6a3f88baa5",
    "updateMode": "selective",
    "controls": {
        "archiveTimestamp": "2021-03-23T17:14:46.030Z"
    }
}
```

**Comment by dstokesf5** [comment]
This has been added to our internal product backlog as  AUTOTOOL-2377. Thank you for reporting it.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 3.29.0 release.
